projects/bp-gallery/src/components/views/contact/ContactFormView.tsx

```diff
@@ -61,7 +61,7 @@ const ContactFormView = () => {
   );
 
   return (
-    <div className='max-w-[1200px] h-full flex flex-col flex-nowrap items-center m-auto bg-white shadow-lg'>
+    <div className='max-w-[1200px] min-h-full flex flex-col flex-nowrap items-center m-auto bg-white shadow-lg'>
       <div className='mx-auto mt-20 p-4'>
```

Comment:
Maybe remove the `mt-20` here, while you're at it, I think the massive amount of space above the form looks weird.